packages/devextreme/js/__internal/grids/grid_core/focus/m_focus.ts

```diff
@@ -863,6 +869,14 @@ const rowsView = (Base: ModuleType<RowsView>) => class RowsViewFocusController e
     }
 
     $row.attr('tabIndex', tabIndex);
+    const rowIndexFromOption = this.option('focusedRowIndex')! - this._dataController.getRowIndexOffset(true);
```

Comment:
Minor
```suggestion
    const focusedRowIndex = this.option('focusedRowIndex')!;
    const rowIndexFromOption = focusedRowIndex - this._dataController.getRowIndexOffset(true);
```

Reply:
Better to hide this intermediate to prevent it from being used in future